ui/common/css/vendor/chessground/_board-2d.scss

```diff
@@ -140,6 +150,8 @@ $togyzkumalak-variants: ('togyzkumalak'); // must use same board
 
 $backgammon-variants: ('backgammon', 'nackgammon'); // must use same board
 
+$breakthrough-variants: ('breakthroughtroyka', 'minibreakthroughtroyka'); // must use same board
```

Comment:
Not sure what this comment was about (copied from line above)

Reply:
the board image must be the same for both variants, which is not the case for breakthrough but depends how its getting used as a variable. This comment was copied from shogi and xiangqi where the board images are different and require different css link to image for them.